editors/kakoune/Portfile

```diff
@@ -0,0 +1,25 @@
+PortSystem          1.0
```

Comment:
Please add the standard modeline, followed by a blank line, before the PortSystem line. See https://guide.macports.org/#development.creating-portfile

Reply:
The guide says optional, but added.